### PR TITLE
added card badges

### PR DESCRIPTION
**Pull request by sohan9819**
Added 4 types of card badges

### CODE REVIEW

**Review by Rutwik87 — APPROVED** [review]

File: css/components/card_badge.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.card_with_badges {
+  height: 36rem;
+  width: 28rem;
+  position: relative;
+  background-image: linear-gradient(to bottom, transparent, var(--color-black)),
+    url("https://source.unsplash.com/random/500x500");
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (DevNirah7) [inline css/components/card_badge.css:8]: Can use shorthand here

**Review by DevNirah7 — APPROVED** [review]
Looks good to commit!

File: css/components/card_badge.css (hunk 1 above)
Comment (sohan9819) [reply css/components/card_badge.css:8]: Yes may be but it would get more confusing so just to keep it simple I did it in this way 😊

**Comment by ashifkhn** [comment]
Looks good,

**Review by rahul31agrawal — COMMENTED** [review]
looks good to me

**Pull request merged**